### PR TITLE
[test] add test for fileutil

### LINKED ISSUE #3113: Add unit test to https://github.com/iotexproject/iotex-core/tree/master/pkg/util/fileutil
Add unit test to enhance robustness

### PR DESCRIPTION
**Pull request by LuckyPigeon**
fix #3113

### CODE REVIEW

File: pkg/util/fileutil/fileutil_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		require.False(t, isFileExists)
+	})
+}
+
+func TestGetFileAbsPath(t *testing.T) {
+	t.Run("get absolute path in blockchain directory", func(t *testing.T) {
+		pwd, err := os.Getwd()
+		require.NoError(t, err)
+		if runtime.GOOS == "windows" {
+			fileAbsPath := GetFileAbsPath("filename")
```
Comment (CoderZhi) [inline pkg/util/fileutil/fileutil_test.go:37]: move out of `if`

**Review by CoderZhi — APPROVED** [review]

**Comment by huof6829** [comment]
please make fmt/lint before create pr

**Comment by huof6829** [comment]
please make fmt/lint before create pr

**Review by Liuhaai — APPROVED** [review]

**Comment by Liuhaai** [comment]
thx for contribution

**Pull request merged**